oak_restricted_kernel/layout.ld

```diff
@@ -64,15 +63,10 @@ SECTIONS {
      */
     . += KERNEL_MEM_START;
 
-    data_start = .;
-    text_start = .;
```

Comment:
Why don't we need these symbols anymore?

Reply:
Ages ago when we were using code from rust_hypervisor_firmware crate the code in there needed those symbols. We've since stopped using that crate, but never cleaned up our linker script.